specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -1227,6 +1233,13 @@
           "type": "array",
           "items": {
             "type": "string"
+            }
+        },
+        "components": {
+          "description": "Components state compliance records. By default this is not populated.",
```

Comment:
>By default this is not populated. [](start = 63, length = 33)

Instead of saying this can you specify when this would be populated?

Reply:
fixed.